openapi/openapi.yaml

```diff
@@ -8,6 +8,10 @@ servers:
   - url: https://zoning.planningdigital.com/api
 tags:
   - name: Boroughs
+  - name: LandUses
```

Comment:
I wouldn't hold up merging over this but can we do spaces in tags? I think it would be ideal to make that as human-friendly as possible. I would rather this render as "Land Uses" than "LandUses" in redoc
![image](https://github.com/NYCPlanning/ae-zoning-api/assets/9055367/f90d9fc7-ea59-417c-9eac-dffcfdb7db51)

Reply:
I'll give it a shot and report back

Reply:
Success